packages/graphql-fixtures/package.json

```diff
@@ -30,7 +30,7 @@
     "graphql-tool-utilities": "^3.0.0"
   },
   "peerDependencies": {
-    "graphql-typed": ">=0.6.1 <2.0.0"
+    "graphql-typed": "^0.6.1 || ^0.7.0 || ^1.0.0 || ^2.0.0"
```

Comment:
The versions tab on https://www.npmjs.com/package/graphql-typed shows that 0.6.x an 0.7.x were the only pre-v1 releases greater that 0.6.1, so this maintains the existing set of range, while adding `^2.0.0`